### PR TITLE
add default resource values for appService

### PR DESCRIPTION
**Pull request by ChuckHend**

### CODE REVIEW

File: charts/tembo-operator/templates/crd.yaml [hunk 1]
```diff
@@ -152,8 +152,14 @@ spec:
                       - readiness
                       type: object
                     resources:
+                      default:
+                        limits:
+                          cpu: '2'
+                          memory: 2Gi
+                        requests:
+                          cpu: 500m
+                          memory: 512Mi
```
Comment (sjmiller609) [inline charts/tembo-operator/templates/crd.yaml:161]: Kinda big default, also I think default should have no oversubscription for memory.

```suggestion
                      default:
                        limits:
                          cpu: '1'
                          memory: 512Mi
                        requests:
                          cpu: 250m
                          memory: 512Mi
```

**Review by sjmiller609 — APPROVED** [review]

File: charts/tembo-operator/templates/crd.yaml (hunk 1 above)
Comment (ChuckHend) [reply charts/tembo-operator/templates/crd.yaml]: good catch. let me regenerate the CRD

Comment (ChuckHend) [reply charts/tembo-operator/templates/crd.yaml]: 100m and 400m for CPU sound good?

**Review by sjmiller609 — APPROVED** [review]

File: charts/tembo-operator/templates/crd.yaml (hunk 1 above)
Comment (sjmiller609) [reply charts/tembo-operator/templates/crd.yaml]: Yes it looks good

**Pull request merged**